### PR TITLE
[Request][Serverless][8.16] GA-ing alert suppression for IM rule, Threshold rule, ML rule, ES|QL rule and New Terms rule

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5897

**ESS**
- [Suppress detection alerts](https://security-docs_bk_5926.docs-preview.app.elstc.co/guide/en/security/master/alert-suppression.html): The note at the start of the page now shows that alert suppression is only in tech preview for the event correlation (EQL) rule type.
- [Create rule](https://security-docs_bk_5926.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html): Removed the tech preview label at the start of the step that describes how to configure alert suppression for indicator match, threshold, ML, ES|QL, and new term rule types. 
- [Create rule API | Optional alert suppression fields](https://security-docs_bk_5926.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#opt-fields-alert-suppression-create): The note at the start of the page now shows that alert suppression is only in tech preview for the event correlation (EQL) rule type.
- [Update rule API | Optional alert suppression fields](https://security-docs_bk_5926.docs-preview.app.elstc.co/guide/en/security/master/rules-api-update.html#opt-fields-alert-suppression-update): Ditto

**Serverless**
- [Suppress detection alerts](https://elastic-dot-co-docs-production-nbj45fbye-elastic-dev.vercel.app/current/serverless/security/alert-suppression): The note at the start of the page now shows that alert suppression is only in tech preview for the event correlation (EQL) rule type.
- [Create rule](https://elastic-dot-co-docs-production-nbj45fbye-elastic-dev.vercel.app/current/serverless/security/rules-create): Removed the tech preview label at the start of the step that describes how to configure alert suppression for indicator match, threshold, ML, ES|QL, and new term rule types.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-844o4n14r-elastic-dev.vercel.app

* Built with commit b46829a929e0aff21a152d9ea18c4833f95f8af0

> Issues? Visit #next-docs in Slack

**Review by jmikell821 — APPROVED** [review]
Changes LGTM!

**Review by jmikell821 — APPROVED** [review]

File: docs/serverless/alerts/alert-suppression.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ status: in review
 <DocCallOut color="warning" title="Requirements and notice">
     - ((ml-cap)) rules have <DocLink slug="/serverless/security/ml-requirements">additional requirements</DocLink> for alert suppression.
 
-    - Alert suppression is in technical preview for threshold, indicator match, event correlation, and new terms rules. The functionality may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+    - Alert suppression is in technical preview for event correlation rules. The functionality may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
```
Comment (yctercero) [inline docs/serverless/alerts/alert-suppression.mdx:15]: @approksiu Do we specify a tier here? Platinum in ESS would be complete tier?

Comment (yctercero) [reply docs/serverless/alerts/alert-suppression.mdx:15]: Kseniia asked me to follow up - looks like it's available in Essentials, so I would say no need to specify a tier?

**Review by yctercero — APPROVED** [review]
Thank you for the updates!

**Pull request merged**